### PR TITLE
Update to latest rust stable

### LINKED ISSUE #9: Rustflags now require alsow +mutable-globals
I spent quiet some time wondering about linker errors, looks like RUSTFLAGS with latest nightly needs also mutable-globals, e.g.

```
-C target-feature=+atomics,+bulk-memory,+mutable-globals
```

See e.g. https://github.com/rustwasm/wasm-bindgen/issues/2487 and the appropriate fix https://github.com/alexcrichton/wasm-bindgen/commit/ad53c1223a09029b533bdbcc98be5d07a12da71a

### PR DESCRIPTION
**Pull request by ImUrX**
i changed wasm-bindgen min version to .74 (rustwasm/wasm-bindgen#2488) because its where the bug is fixed and we are using the nightly hash of [Rust 1.54](https://github.com/rust-lang/rust/blob/master/RELEASES.md)

closes #9

### CODE REVIEW

**Comment by ImUrX** [comment]
@googlebot I signed it!

**Comment by RReverser** [comment]
Wow, that was quick. Thanks! I'll take a look a little later.

**Comment by RReverser** [comment]
Thank you!

**Comment by RReverser** [comment]
Hmm this looks bad. https://github.com/GoogleChromeLabs/wasm-bindgen-rayon/runs/3254179053?check_suite_focus=true

File: demo/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "build:wasm-st": "wasm-pack build --target web",
-    "build:wasm-mt": "cross-env RUSTFLAGS='-C target-feature=+atomics,+bulk-memory' rustup run nightly-2021-02-11 wasm-pack build --target web --out-dir pkg-parallel -- --features parallel -Z build-std=panic_abort,std",
+    "build:wasm-mt": "cross-env RUSTFLAGS='-C target-feature=+atomics,+bulk-memory' rustup run nightly-2021-07-29 wasm-pack build --target web --out-dir pkg-parallel -- --features parallel -Z build-std=panic_abort,std",
```
Comment (RReverser) [inline demo/package.json:5]: Looks like it's due to missed `mutable-globals` feature here.

Comment (RReverser) [reply demo/package.json:5]: Let's see... https://github.com/GoogleChromeLabs/wasm-bindgen-rayon/commit/24b9456bf7a3b6f14b6505d8c66f2c457d69ef79

Comment (RReverser) [reply demo/package.json:5]: Yeah that was it.

**Comment by ImUrX** [comment]
Sorry didnt notice, good to know that its now in latest rust :P

**Pull request merged**